proto/bootz.proto

```diff
@@ -147,6 +162,13 @@ message BootstrapStreamRequest {
 // GetBootstrapDataRequest based on the identity message.
 message BootstrapStreamResponse {
   message Challenge {
+    // TPM 2.0 without IDevId may be provisioned using using EK or PPK
+    enum TPM20CsrRequest {
+       CSR_REQUEST_UNKNOWN = 0;
```

Comment:
Also, there's apparently a weird name of spaces before these values. I think you'll need to delete 1 or 2 spaces.